projects/components/src/filtering/filter/filter-operators.ts

```diff
@@ -9,6 +9,12 @@ export const enum FilterOperator {
   GreaterThanOrEqualTo = '>=',
   Like = '~',
   In = 'IN',
+  /**
+   * The difference between 'Contains' and 'Like' is that 'Like' will be treated as a regex filter, not escaping
```

Comment:
nit: I would get list into impl details here. CONTAINS is a filter if the field contains the provided string LIKE is a regex match filter.

Reply:
Please follow up on this if you agree